src/Tests/ObjectLifetimeTests/ObjectLifetimeTests.Lifted/ObjectLifetimeTests.Lifted (Package)/ObjectLifetimeTests.Lifted (Package).wapproj

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Are any of the files in the "(Package)" directory needed? I thought these tests worked without using the WAP?

Reply:
They do, I wasn't sure whether or not the WAP was still necessary to get them running. Tested this locally and the tests succeed without the wapproj-related items so I've removed them.